tests/e2e/k8s/test_basic.go

```diff
@@ -28,6 +31,74 @@ import (
 	"github.com/clusterlink-net/clusterlink/tests/e2e/k8s/util"
 )
 
+func (s *TestSuite) TestConnectivityCRD() {
+	s.RunOnAllDataplaneTypes(func(cfg *util.PeerConfig) {
+		cfg.CRDMode = true
+		cl, err := s.fabric.DeployClusterlinks(2, cfg)
+		require.Nil(s.T(), err)
+
+		require.Nil(s.T(), cl[0].CreateService(&httpEchoService))
+		require.Nil(s.T(), cl[0].CreateExport("dontcare", &httpEchoService))
+		// TODO: create policy
+		require.Nil(s.T(), cl[1].CreatePeer(cl[0]))
+
+		importedService := &util.Service{
+			Name: httpEchoService.Name,
+			Port: 80,
+		}
+		require.Nil(s.T(), cl[1].Cluster().Resources().Create(
```

Comment:
This code in the end will move to createImport?
Maybe is better to create it already and call the function createImportCRD and later change the name?